generators/server/templates/_gradle.properties

```diff
@@ -13,6 +13,8 @@ cassandra_unit_spring_version=3.0.0.1
 <%_ if (cucumberTests) { _%>
 cucumber_version=1.2.4
 <%_ } _%>
+# Overridden to get metrics-jcache
+dropwizard-metrics.version=3.2.2
```

Comment:
I think it need to be rename to: `dropwizard_metrics_version=3.2.2`

And this [file](https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/_build.gradle#L196) needs this:
```groovy
    <%_ if (hibernateCache === 'ehcache') { _%>
    compile "io.dropwizard.metrics:metrics-jcache:${dropwizard_metrics_version}"
    <%_ } _%>
```